Extension/src/LanguageServer/settings.ts

```diff
@@ -477,7 +477,9 @@ export class CppSettings extends Settings {
         }
         return val as string;
     }
-
+    public get copilotCompletionFeatureFlag(): string | undefined {
+        return super.Section.get<any>("copilotCompletionFeatureFlag") as string | undefined;
```

Comment:
@lukka Is this setting supposed to be added to the package.json or it intentionally a "hidden" setting?

Reply:
@sean-mcmanus correct, it is not meant to be public :)